### DISCUSSION TITLE
<Issues related to git bash>

### DISCUSSION THREAD

**Discussion by Sowjanya2527** [community/community#124064] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

#23204 My self Sowjanya I am new to the GitHub, I am trying my level best to learn GitHub.
# Here is my issue 
- Actually now I am working with the Git, I am unable push my change in the repository It's asking me to login how can I do so

**Comment by Chittu13** [comment]
- you can use this commands in the Git bash 
```git config --global user.name "Username"```
```git config --global user.email "email@email.com"```
